spec/models/food_spec.rb

```diff
@@ -0,0 +1,45 @@
+# spec/models/food_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  let(:user) { create(:user) }
+
+  it 'is valid with valid attributes' do
+    food = build(:food, user: user)
+    expect(food).to be_valid
+  end
+
+  it 'is not valid without a name' do
+    food = build(:food, user: user, name: nil)
+    expect(food).to_not be_valid
+  end
```

Comment:
- [ ] There are errors raised while running the spec files. Please fix that.

![image](https://github.com/diarisdiakite/Recipe-app/assets/68796232/5618e632-59ae-429c-9b59-4aae353f837f)

Reply:
Hi @Omar-Muhamad 
Thank you for the review🙏
but the tests are working I test it 3 times.

![Screenshot 2023-11-18 060742](https://github.com/diarisdiakite/Recipe-app/assets/126192888/f059e960-32bd-4fe1-885d-00e7ffe699cd)